index.ts

```diff
@@ -1,54 +1,31 @@
 import { multiUrlGenerator } from './lib/multiUrlGenerator';
-import readline from 'readline';
+import fetch from 'node-fetch';
 
-const clientUI = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout,
-});
+// Test fetching Wiki articles
+const requestUrls = multiUrlGenerator(
```

Comment:
I created a simple use case to test the main program with 2 wiiki articles in 2 different languages.

Reply:
👍